frontend/app/work_packages/services/work-package-field-service.js

```diff
@@ -218,8 +218,8 @@ module.exports = function(
       return true;
     }
 
-    // strategy pattern, guys
-    if (field === 'spentTime' && WorkPackageFieldService.getValue(workPackage, field) === 'PT0S') {
+    if (field === 'spentTime' && WorkPackageFieldService.getValue(workPackage, field) === 'PT0S'
+        || value === '0 hours') {
```

Comment:
I think you cannot compare to `0 hours` as this string maybe localized and you will get `0 Stunden` or sth. else. Check by changing your personal language setting in the personal profile.